github-dark.user.css

```diff
@@ -10281,18 +10281,18 @@
   body[class="page-responsive"] .btn-primary[aria-selected=true] {
     background-color: #595;
   }
-  /* github-mobile: "hsla(0,0%,100%,.8)", "#94d3a2", "rgba(27,31,35,.1)", "rgba(27,31,35,.1)" */
+  /* github-mobile: "color: hsla(0,0%,100%,.8)", "#94d3a2", "rgba(27,31,35,.1)", "rgba(27,31,35,.1)" */
   body[class="page-responsive"] .btn-primary.disabled,
   body[class="page-responsive"] .btn-primary:disabled,
   body[class="page-responsive"] .btn-primary[aria-disabled=true] {
-    color: hsla(0, 0%, 100%, .8);
+    color: hsla(0, 0%, 100%, .3);
     background-color: #040;
     border-color: rgba(220, 220, 220, .1);
     box-shadow: 0 1px 0 #000, inset 0 1px 0 hsla(0, 0%, 100%, .03);
   }
-  /* github-mobile: "hsla(0,0%,100%,.8)" */
+  /* github-mobile: "color: hsla(0,0%,100%,.8)" */
   body[class="page-responsive"] .btn-primary .octicon {
-    color: hsla(0, 0%, 100%, .8);
+    color: hsla(0, 0%, 100%, .3);
```

Comment:
Here at this value the octicon is going to break because `$color` thinks this is OK. SO we fix one and break others.